README.md

```diff
@@ -10,6 +10,10 @@ SYNOPSIS
     easy-stage-dataset -t <submission-timestamp> -u <urn> -d <doi> [ -o ] \
                           <EASY-bag> <staged-digital-object-set>
 
+    easy-stage-file-item [<options>...] <staged-digital-object-set>
+    easy-stage-file-item <staged-digital-object-set> <csv-file>
```

Comment:
It woudl be more intuitive to put the csv-file before the staged-digital-object-set. This is the pattern seen in many command line tools that take an input and output specification argument (e.g., `cp`)